subsys/bluetooth/Kconfig

```diff
@@ -6,8 +6,8 @@
 
 menuconfig BT
 	bool "Bluetooth"
-	# Some BT threads use co-op priorities to implement critical sections,
-	# will need some refactoring to work on SMP systems.
+#Some BT threads use co - op priorities to implement critical sections,
+#will need some refactoring to work on SMP systems.
```

Comment:
Any reason to change the alignment here? We keep comments aligned along with the other attributes of the Kconfig usually.

Reply:
clang-format >.>